.github/workflows/swift.yaml

```diff
@@ -0,0 +1,45 @@
+name: Swift - Workflow - Cocoapods (Xcode 11.2.1)
+
+on: [push]
```

Comment:
You probably want to include a file filter so you don't run this job for Android PRs. See this doc: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths